fastlane/Fastfile

```diff
@@ -613,10 +534,9 @@ platform :ios do
     commit_version_bump
```

Comment:
Won't this `commit_version_bump` need to be pushed?

Reply:
@iangmaia should be fixed in 34b70860d

_(I know, I used `after_confirming_push` while I plan to remove it 😅 … but we'll revisit that in a separate PR)_